packages/teams-js/src/public/interfaces.ts

```diff
@@ -82,6 +82,24 @@ export interface TabInstance {
    * Website URL of this tab
    */
   websiteUrl?: string;
+
+  /**
+   * AppId of this tab
+   */
+  appId?: string;
```

Comment:
>string [](http://example.com/codeflow?start=10&length=6)

Can you use the `AppId` type for this, rather than a string?

Reply:
I think we should keep it string to be compatible with other APIs. It is a string in all other APIs as well as in the TMP implementation. I see there is a class AppId, but I don't see it being used anywhere.